aiosmtpd/context.py

```diff
@@ -0,0 +1,15 @@
+class SessionContext:
```

Comment:
Totaly agree with renaming. I like attrs idea, just aware of some reasons:
1. I don`t want add dependency for every small thing. I can became nodejs like dependences hell. 
2. I keep in mind that aiosmtpd can get in stdlib, so it must not depend on anything that is not in stdlib. If get it away, I prefer to write tests on pytest, not unittesting module.
3. aiosmtpd idea is quite simple and it is naturally core thing itself. I feel that it is simple and core enough to be written without any external non stdlib dependencies.

But with attrs it really will be more pretty, agree with that. If you think it will be better, I`ll include it.